crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -340,29 +341,33 @@ impl CumulocityConverter {
 
     pub async fn process_health_status_message(
         &mut self,
+        entity: &EntityTopicId,
         message: &Message,
     ) -> Result<Vec<Message>, ConversionError> {
         let mut mqtt_messages: Vec<Message> = Vec::new();
 
         // Send the init messages
-        if check_tedge_agent_status(message)? {
-            create_tedge_agent_supported_ops(self.ops_dir.clone()).await?;
+        // TODO: use entity store to check service statuses
+        if is_message_tedge_agent_up(message)? {
```

Comment:
The health status could arguably be a part of an entity store, since it's supposed to know about all entities anyway. I'd be for deferring it to another PR though.

Reply:
This will be replaced by a check of the software-list capability.